### PR TITLE
Shaving Corgis (more useless corgi stuff)

### PR DESCRIPTION
**Pull request by Niknakflak**
Adds the ability to shave all corgis.  (Includes, Ian, Lisa, Generic Corgi, and puppies.)  
Because the sprites were there and I wanted it, so YEA

### CODE REVIEW

**Comment by ike709** [comment]
No shaving Ian. Ever.

File: code/modules/mob/living/simple_animal/friendly/corgi.dm [hunk 1]
```diff
@@ -33,6 +35,21 @@
...
 
+/mob/living/simple_animal/corgi/attackby(var/obj/item/O as obj, var/mob/user as mob)
+	if(user.stat == CONSCIOUS && istype(O, /obj/item/weapon/razor))
+		if (shaved == 1)
+			user << "<span class='warning'>You can't shave this corgi, it's already been shaved.</span>"
+			return
+		user.visible_message("<span class='notice'>[user] starts to shave [src] using \the [O].</span>")
+		if(do_after(user, 50))
+			user.visible_message("<span class='notice'>[user] shaves [src]'s hair using \the [O]. </span>")
+			playsound(loc, 'sound/items/Welder2.ogg', 20, 1)
```
Comment (RemieRichards) [inline code/modules/mob/living/simple_animal/friendly/corgi.dm:46]: > Welder2

umm.

Comment (Niknakflak) [inline code/modules/mob/living/simple_animal/friendly/corgi.dm:46]: Its the same sound for shaving humans.

Comment (RemieRichards) [inline code/modules/mob/living/simple_animal/friendly/corgi.dm:46]: is it Really? Oh right then.

**Comment by ike709** [comment]
NO. SHAVING. IAN.

**Pull request merged**